react-native/react-native-flipper/android/src/debug/java/com/facebook/flipper/reactnative/ReactNativeFlipper.java

```diff
@@ -5,7 +5,7 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-package com.reactnativeflipperexample;
```

Comment:
I have a feeling that we had that for some reason. Did it build fine for you after changing this?

Reply:
Yes, I ran on my phone and it worked. I will test later on Android Emulator to make sure if its working properly

it's being used [here](https://github.com/facebook/flipper/blob/24527300d42e55cbd7716814c99f5cbaad20d181/react-native/ReactNativeFlipperExample/android/app/src/main/java/com/reactnativeflipperexample/MainApplication.java#L72)

So I changed this file to use `initializeFlipper` from `react-native-flipper` added in this PR.
But the developer can use a customized initialization method, just need to create a custom Java class (like the current ReactNative template does)